specs/fault-dispute-game.md

```diff
@@ -69,9 +72,12 @@ We refer to this state as the **ABSOLUTE\_PRESTATE**.
 
 ### Claims
 
-Claims assert an execution trace. This is represented as `ClaimHash`, a `bytes32` commitment to
-the last VM state in a trace. A FDG is initialized with a root claim, which commits to the entire
-execution trace. As we'll see later, there can be multiple claims, committing to different states in the FDG.
+Claims assert an [output root][g-output-root] or the state of the FPVM at a given instruction. This is represented as
+`ClaimHash`, a `bytes32` representing either an [output root][g-output-root] or a commitment to the last VM state in a
+trace. A FDG is initialized with an output root that corresponds to the state of L2 at a given L2 block number, and
+execution trace subgames are initialized with a claim that commits to the entire execution trace between two conseuctive
```

Comment:
```suggestion
execution trace subgames at `SPLIT_DEPTH + 1` are initialized with a claim that commits to the entire execution trace between two consecutive
```

Otherwise, this contradicts the new definition of subgames (1-deep DAGs) when the subgame is rooted at SPLIT_DEPTH+2 for example.